.github/workflows/pylint.yml

```diff
@@ -44,10 +44,10 @@ jobs:
       - name: Checkout PR branch
         uses: actions/checkout@v3
```

Comment:
An unrelated issue, but should this be updated to `actions/checkout@v4`?

Reply:
Good point, thanks for catching it.